src/transformers/models/llama/configuration_llama.py

```diff
@@ -64,6 +64,13 @@ class LlamaConfig(PretrainedConfig):
             relevant if `config.is_decoder=True`.
         tie_word_embeddings(`bool`, *optional*, defaults to `False`):
             Whether to tie weight embeddings
+        rope_scaling (`Dict`, *optional*):
+            Experimental feature -- dictionary containing the scaling configuration for the RoPE embeddings. Currently
```

Comment:
I would put "Experimental feature" as a warning at the end of the docstring.

<Tip warning={true}>

This is an experimental feature, subject to breaking API changes in future versions.

</Tip>